programming4.cpp

```diff
@@ -82,7 +82,14 @@ int main()
         std::cout << vector4[i-1] << std::endl;
     }
 
-
+    std::vector<double> vector5;
+    std::cout << "This is vector 5" << std::endl;
+    // for(size_t i = 1; vector4.size(); ++i)
+    // {
+    //     vector5.push_back((vector4[i] - vector4[i -1]) / (vector1[i + 2] - vector1[i -1]));
+    //     std::cout << vector5[i] << std::endl;
```

Comment:
```suggestion
    //    std::cout << vector5[i-1] << std::endl;
```
Vector 5 only has one element. Going out of bounds!